### ISSUE TITLE
OpenSSL::SSL::SSLContext#min_version= failure

### ISSUE THREAD

**Issue by djberg96** [jruby/jruby#6409]
* jruby 9.2.13.0 (2.5.7) 2020-08-03 9a89c94bcc OpenJDK 64-Bit Server VM 14.0.2+12-46 on 14.0.2+12-46 +jit [darwin-x86_64]
* Mac (Catalina)

I'm currently seeing a bug attempting to set the minimum TLS version in an openssl security context. This came up recently due to some changes between openssl 1.0 and 1.1. Below is a minimal script that duplicates the issue:

```
require 'net/smtp'

context = Net::SMTP.default_ssl_context
context.min_version = OpenSSL::SSL::TLS1_VERSION
```

The result is:

```
NoMethodError: undefined method `set_minmax_proto_version' for #<OpenSSL::SSL::SSLContext:0x5d425813 @max_proto_version=nil>
  min_version= at /Users/dberger/.rbenv/versions/jruby-9.2.13.0/lib/ruby/stdlib/jopenssl23/openssl/ssl.rb:175
        <main> at jruby_smtp_bug.rb:4
```

With C Ruby, this works as expected.

**Comment by MariuszCwikla** [comment]
It's a bug in jruby-openssl. I've opened PR jruby/jruby-openssl#215 that attempts to fix it.

**Comment by headius** [comment]
Thank you @MariuszCwikla! I see @kares already reviewed your PR so it looks like it's on its way.

**Comment by headius** [comment]
Marked as non-release but once fixed we can pull the new jruby-openssl into JRuby 9.2.14 or 9.3.

**Comment by prakashguru** [comment]
@headius Seems this issue is fixed and available in jruby-openssl 0.10.7 release. Can this be pulled into jruby?
